src/combatFocus.js

```diff
@@ -83,52 +96,53 @@ Hooks.on("renderActorSheet", (app, html, data) => {
         }
 
     })
-})
+});
+
 let delay = 5;
-if (typeof ForgeVTT !== 'undefined') delay = 15
+if (typeof ForgeVTT !== 'undefined') delay = 15;
 
 Hooks.on("updateCombat", async (combat, changed, options, userId) => {
 
-    if (!("turn" in changed) && changed.round !== 1) {
-        return;
-    }
+    if (!("turn" in changed) && changed.round !== 1) return;
+    if (game.combats.get(combat.id).data.combatants.length == 0) return;
 
-    const firstGm = game.users.find((u) => u.isGM && u.active);
-    if (firstGm && game.user === firstGm) {
+    const combatFocusPostion = game.settings.get('Next-Up', 'combatFocusPostion');
+    const combatFocusPan = game.settings.get('Next-Up', 'combatPan');
+    const closeWhich = game.settings.get('Next-Up', 'closewhich');
+    const closeType = game.settings.get('Next-Up', 'closetype');
+    const combatFocusType = game.settings.get('Next-Up', 'combatFocusType');
+    const playerPan = game.settings.get('Next-Up', 'playerPan');
+    let currentWindows = Object.values(ui.windows);
 
-        if (game.combats.get(combat.id).data.combatants.length == 0) return;
+    let nextTurn = combat.turns[changed.turn];
+    if (changed.turn === undefined) nextTurn = combat.turns[0]
+    const previousTurn = combat.turns[changed.turn - 1 > -1 ? changed.turn - 1 : combat.turns.length - 1]
 
+    let nextTokenId = null;
+    if (getProperty(nextTurn, "tokenId")) {
+        nextTokenId = nextTurn.tokenId;
+    }
+    else {
+        nextTokenId = getProperty(nextTurn, token._id);
+    }
 
-        let nextTurn = combat.turns[changed.turn];
-        if (changed.turn === undefined) nextTurn = combat.turns[0]
-        const previousTurn = combat.turns[changed.turn - 1 > -1 ? changed.turn - 1 : combat.turns.length - 1]
+    let currentToken = canvas.tokens.get(nextTokenId);
+    let previousToken = canvas.tokens.get(previousTurn.tokenId)
 
-        let nextTokenId = null;
-        if (getProperty(nextTurn, "tokenId")) {
-            nextTokenId = nextTurn.tokenId;
-        }
-        else {
-            nextTokenId = getProperty(nextTurn, token._id);
-        }
+    if (!currentToken.actor) {
+        return;
+    }
 
-        let currentToken = canvas.tokens.get(nextTokenId);
-        let previousToken = canvas.tokens.get(previousTurn.tokenId)
 
-        if (!currentToken.actor) {
-            return;
-        }
 
-        const combatFocusPostion = game.settings.get('Next-Up', 'combatFocusPostion')
-        const combatFocusPan = game.settings.get('Next-Up', 'combatPan')
-        const closeWhich = game.settings.get('Next-Up', 'closewhich')
-        const closeType = game.settings.get('Next-Up', 'closetype')
-        const combatFocusType = game.settings.get('Next-Up', 'combatFocusType')
-        let currentWindows = Object.values(ui.windows)
+    await sleep(delay);
+    await currentToken.control();
```

Comment:
Will this not throw errors on the player side when trying to controll non-owned tokens?

Reply:
That's a good point, though no errors came up on the players' side for me. My guess is the method just tries to control the token, sees that it's not owned, then gives up without a fuss.
You could definitely move it into the GM block to be safe, e.g. move it to line 142.

Reply:
Changed in new commit